shex/patientConsentShape.shex

```diff
@@ -0,0 +1,51 @@
+PREFIX : <http://purl.org/ejp-rd/cde/v030/shex/>
+PREFIX obo: <http://purl.obolibrary.org/obo/> 
+PREFIX sio: <http://semanticscience.org/resource/>
+PREFIX xsd: <http://www.w3.org/2001/XMLSchema#>
+PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#>
+
+:identifierShape IRI {
+    a [sio:SIO_000115] ;
+    sio:SIO_000020 @:consentRoleShape ;
+    sio:SIO_000300 xsd:string
+}
+
+:personShape IRI { 
+    a [sio:SIO_000498] ;
+    sio:SIO_000228 @:consentRoleShape ;
+    sio:SIO:000008 @:consentAttributeShape
+}
+
+:consentRoleShape IRI {
+    a [obo:OBI_0000093] ;
+    a [sio:SIO_000016] ;
+    rdfs:label xsd:string;
+    sio:SIO_000356 @:consentProcessShape
+}
+
+:consentProcessShape IRI {
+    a [obo:OBI_0000810] ;
+    sio:SIO_000669 xsd:date ;
+    sio:SIO_000229 @:patientconsentOutputShape ;
+    sio:SIO_000230 @:consentOutputShape
+}
+
+:patientconsentOutputShape IRI {
+    a [sio:SIO_000015] ;
+    rdfs:label xsd:string ;
+    sio:SIO_000669 xsd:date ;
+    sio:SIO_000300 xsd:string ;
+    sio:SIO_000628 @:consentAttributeShape
+}
+
+:consentOutputShape IRI {
```

Comment:
According to `sio:SIO_000230` （sio:has-input) in `consentProcessShape`, this is the input instead of output of an consent. It would be great if you can change the name as "input". This also applied to Turtle file @rajaram5.